Final tweaks to mobile wallet switcher

akalin-keybase opened the pull request:
- Set max-height to 95%.
- Fix off-by-one bug where we'd always have a scrollbar.
- Fix weird centering of account name.
- Fix row heights to match design.
- Make 'What is Stellar?' fixed.

akalin-keybase commented on shared/common-adapters/list.native.js:
```diff
@@ -32,6 +32,14 @@ class List extends PureComponent<Props<any>, void> {
       <View style={collapseStyles([styles.outerView, this.props.style])}>
         {/* need windowSize so iphone 6 doesn't have OOM issues */}
+        {/* We can use
+            initialScrollIndex={this.props.fixedHeight ? this.props.selectedIndex : undefined}
```
Ideally we'd center the scroll view so that the selected wallet is visible, but that didn't work well with indices close to the end.

akalin-keybase commented on shared/wallets/wallet/header/wallet-switcher/index.native.js:
```diff
@@ -114,6 +116,9 @@ export const WalletSwitcher = (props: Props) => {
   ]
 
+  // Menu items plus pinned cancel item on bottom.
+  const height = rowHeight * (menuItems.length + 1)
```
this was off by one, so even if you had two wallets you'd have to scroll to see the other one.

akalin-keybase commented on shared/wallets/wallet/header/wallet-switcher/index.native.js:
```diff
@@ -148,7 +154,7 @@ export const WalletSwitcher = (props: Props) => {
   container: {
     justifyContent: 'flex-end',
-    maxHeight: '50%',
+    maxHeight: '100%',
```
design has 100%, and @cecileboucheron confirmed that's what's desired, so just using that

akalin-keybase commented on shared/wallets/wallet/header/wallet-switcher/wallet-row/index.js:
```diff
@@ -58,7 +58,7 @@ const WalletRow = (props: Props) => {
       {/* Just needed for proper centering */ emptyIcon}
       <Kb.Box2 direction="vertical" style={styles.rowContainer}>
-        <Kb.Box2 direction="horizontal" fullWidth={true}>
+        <Kb.Box2 direction="horizontal">
```
fixes weird centering of account name

akalin-keybase commented:
![simulator screen shot - iphone 6 - 2019-01-08 at 12 44 31](https://user-images.githubusercontent.com/13270622/50851789-6208ef00-1343-11e9-8711-f647f4cc3341.png)

cecileboucheron commented:
@akalin Could we leave some space for the status bar though? 
![image](https://user-images.githubusercontent.com/2807426/50859129-54edff00-1347-11e9-8a72-8d09e0980528.png)

akalin-keybase commented:
Okay, @cecileboucheron I addressed all your comments. PTAL!

![image](https://user-images.githubusercontent.com/13270622/50941536-2b61d000-143a-11e9-88b6-8ce853e80008.png)

akalin-keybase commented:
Okay, fixed some stuff noticed by @cecileboucheron 

![image](https://user-images.githubusercontent.com/13270622/50995770-e5a61580-14d4-11e9-9b14-f69ed5052f4b.png)

akalin-keybase commented:
@cecileboucheron addressed all your comments. I'll merge this now, I can address any follow-up issues separately.
